### PR TITLE
Fix throw wrong exception in `CreateTableTask`

### PR DESCRIPTION
**Pull request by polaris6**
## Description
Fix throw wrong exception in `CreateTableTask`.

## Documentation
(x) No documentation is needed.

## Release notes
(x) No release notes entries required.

### CODE REVIEW

File: core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java [hunk 1]
```diff
@@ -132,7 +132,7 @@ ListenableFuture<Void> internalExecute(CreateTable statement, Session session, L
         }
         catch (TrinoException e) {
             if (e.getErrorCode().equals(UNSUPPORTED_TABLE_TYPE.toErrorCode())) {
-                throw semanticException(TABLE_ALREADY_EXISTS, statement, "Table '%s' of unsupported type already exists", tableName);
+                throw new TrinoException(UNSUPPORTED_TABLE_TYPE, format("The type of table '%s' is unsupported", tableName));
```
Comment (ebyhr) [inline core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java:135]: The original exception seems better. What's the motivation of this change?

Comment (polaris6) [reply core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java:135]: Because I thought the exception should not be `TABLE_ALREADY_EXISTS`, but it looks like I misunderstood, I'll turn off this pr.
